.happy/terraform/modules/sfn_config/gisaid.wdl

```diff
@@ -175,8 +175,12 @@ task TransformGISAID {
     aspen_creation_rev=$COMMIT_SHA
 
     # fetch aspen config
+    set +x  # don't echo secrets
+    echo "* set \$genepi_config"
```

Comment:
I don't understand the `*` purpose? Is it there to indicate that something has been redacted? I think a single asterisk isn't very clear for that. I'd either use `*******` or `XXXXXXX`, more symbols. But I think it's probably clearer to just echo "Setting \$genepi_config" or maybe if you want to be really explicit about why we're doing the approach while printing out the value "Setting \$genepi_config var (not printing value because contains secrets)" or something.

Reply:
ok i'll update the wording